build

```diff
@@ -22,13 +22,44 @@ require(){ command -v $1 > /dev/null 2>&1 || { suggestion=""; if [ ! -z "$2" ];
 if [ $# -eq 1 ]; then if [ "$1" = "--help" -o "$1" = "-h" ]; then usage; fi fi
 #End boilerplate
 
+# detect platform
+platform="$(uname -s)"
+
+if [ "$platform" = "Darwin" ]; then
+    installMsg="brew install"
+else
+    installMsg="apt-get install"
+fi
+
+
 #Check for dependencies
-require "xsltproc" "Try: apt-get install xsltproc"
-require "xmllint" "Try: apt-get install libxml2-utils"
-require "xmlstarlet" "Try: apt-get install xmlstarlet"
-require "xpath" "Try: apt-get install libxml-xpath-perl"
-require "mogrify" "Try: apt-get install imagemagick"
-require "zip" "Try: apt-get install zip"
+require "xsltproc" "Try: ${installMsg} xsltproc"
+require "xmllint" "Try: ${installMsg} libxml2-utils"
+require "xmlstarlet" "Try: ${installMsg} xmlstarlet"
+require "xpath" "Try: ${installMsg} libxml-xpath-perl"
+require "mogrify" "Try: ${installMsg} imagemagick"
+require "zip" "Try: ${installMsg} zip"
+require "python3" "Try: ${installMsg} python3"
+require "rsvg-convert" "Try: ${installMsg} librsvg"
+
+if [ "$platform" = "Darwin" ]; then
+	require "gsed" "Try: ${installMsg} gnu-sed"
+fi
+
+# configure for current platform
+case "$platform" in
+	Darwin)
+		sedCommand="$(which gsed)"
+		realpath(){ pushd . > /dev/null; if [ -d "$1" ]; then cd "$1"; dirs -l +0; else cd "`dirname \"$1\"`"; cur_dir=`dirs -l +0`; if [ "$cur_dir" == "/" ]; then echo "$cur_dir`basename \"$1\"`"; else echo "$cur_dir/`basename \"$1\"`"; fi; fi; popd > /dev/null; }
+		doxpath(){ xpath "$2" "$1" 2> /dev/null; }
+		;;
+
+	*)
+		sedCommand="$(which sed)"
+		doxpath() { xpath -q -e "$1" "$2"; }
```

Comment:
I put `-q` on every `xpath` while you only had it on one. Not sure if that will break anything...